content/ko/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -242,11 +230,48 @@ v1 API의 `zone` 필드로 접근할 수 있다.
 엔드포인트슬라이스 변경의 특성으로 인해, 엔드포인트는 동시에 둘 이상의
 엔드포인트슬라이스에 표시될 수 있다. 이는 다른 엔드포인트슬라이스 오브젝트에
 대한 변경 사항이 다른 시간에서의 쿠버네티스 클라이언트 워치(watch)/캐시에
-도착할 수 있기 때문에 자연스럽게 발생한다. 엔드포인트슬라이스를 사용하는 구현은
-엔드포인트가 둘 이상의 슬라이스에 표시되도록 할 수 있어야 한다. 엔드포인트
-중복 제거를 수행하는 방법에 대한 레퍼런스 구현은 `kube-proxy` 의
+도착할 수 있기 때문에 자연스럽게 발생한다.
+
+{{< note >}}
+엔드포인트슬라이스 API의 클라이언트는 반드시 서비스에 연관된 모든 존재하는 엔드포인트슬라이스를
+반복하고 유일한 네트워크 엔드포인트의 완전한 리스트를 구성해야 한다. 엔드포인트는 다른
+엔드포인트슬라이스에서 중복될 수 있으므로 이를 강조하는 것이 중요하다.
+
+엔드포인트 집계와 중복 제거 방법에 대한 레퍼런스 구현은 `kube-proxy` 의
 `EndpointSliceCache` 구현에서 찾을 수 있다.
+{{< /note >}}
+
+## 엔드포인트와 비교 {#사용동기}
+
+기존 엔드포인트 API는 쿠버네티스에서 네트워크 엔드포인트를 추적하는
+간단하고 직접적인 방법을 제공했다. 쿠버네티스 클러스터와
+{{< glossary_tooltip text="서비스" term_id="service" >}}가 더 많은 수의 백엔드 파드로
+더 많은 트래픽을 처리하고 전송하는 방향으로 성장함에 따라, 이 API의 한계가 더욱 눈에 띄게
+되었다.
+특히나, 많은 수의 네트워크 엔드포인트로 확장하는 것에
+어려움이 있었다.
+
+서비스에 대한 모든 네트워크 엔드포인트가 단일 엔드포인트
+객체에 저장되기 때문에 이러한 엔드포인트 객체들이 상당히 커질 수 있었다. 안정적인
```

Comment:
(메모)
원문에서는 'endpoint(s)' 와 'Endpoints' 로 구분이 되는데
한국어로 둘다 '엔드포인트' 로 기재하니 구분이 안 되네요.. ㅎㅎ

[원문]
> Since all network endpoints for a Service were stored in a single Endpoints object, those Endpoints objects could get quite large.

(이 문장 원문에서는 'Endpoints' 는 모두 'Endpoints object' 로 되어 있어서
번역에도 '엔드포인트 객체' 로 표현되므로 구분이 되는데
이 문서의 다른 문장에서는 그렇지 못한 부분도 있을 것 같습니다.. ㅎㅎ
'Endpoints' 관련 문서의 한국어화 시에는 어쩔 수 없는 것 같기도 하네요.. ㅎㅎ)

Reply:
```suggestion
서비스에 대한 모든 네트워크 엔드포인트가 단일 엔드포인트
객체에 저장되기 때문에 이러한 엔드포인트 객체들이 상당히 커지는 경우도 있었다. 안정적인
```